apps/condo/domains/marketplace/schema/Invoice.js

```diff
@@ -133,16 +139,28 @@ const Invoice = new GQLListSchema('Invoice', {
 
     },
     hooks: {
-        validateInput: ({ resolvedData, operation, existingItem, context }) => {
+        validateInput: async ({ resolvedData, operation, existingItem, context }) => {
             const nextData = { ...existingItem, ...resolvedData }
             if (!((!!nextData.property && !!nextData.unitType && !!nextData.unitName) || !!nextData.accountNumber)) {
                 throw new GQLError(ERRORS.NO_RECEIVERS, context)
             }
 
             if (operation === 'update' && existingItem.status === INVOICE_STATUS_PAID) {
-                // TODO (DOMA-7447) prevent updating only for paid online (with Payment model) +TESTS
+                // Prevent updating only for paid online invoices (with Payment model)
                 // In the case of cash it's may be needed to uncheck the "paid" status
-                throw new GQLError(ERRORS.ALREADY_PAID, context)
+                const payments = await Payment.getAll(
```

Comment:
why getAll if you use id query?

Reply:
There is no constraint "1 payment - 1 invoice". I am searching for payment by `invoice.id`. I'd like to find any payments for the given invoice.